packages/vega-typings/types/spec/encode.d.ts

```diff
@@ -81,27 +81,75 @@ export interface ColorHCL {
   l: NumericValueRef;
 }
 export interface GradientStop {
+  /**
+   * The offset fraction for the color stop, indicating its position within the gradient.
+   */
   offset: number;
-  color: string;
+  /**
+   * The color value at this point in the gradient.
+   */
+  color: Color;
 }
 export interface GradientLinear {
+  /**
+   * The type of gradient. Use `"linear"` for a linear gradient.
+   */
   gradient: 'linear';
+  /**
+   * An array of gradient stops defining the gradient color sequence.
+   */
   stops: GradientStop[];
   id?: string;
+  /**
+   * The starting x-coordinate, in normalized [0, 1] coordinates, of the linear gradient (default 0).
```

Comment:
Better consistently use `__Default value:__`?  Same for other ones with defaults.

Reply:
Done